t/t3400-rebase.sh

```diff
@@ -406,4 +406,18 @@ test_expect_success 'refuse to switch to branch checked out elsewhere' '
 	test_i18ngrep "already checked out" err
 '
 
+test_expect_success MINGW,SYMLINKS_WINDOWS 'rebase when .git/logs is a symlink' '
+	git checkout main &&
+	mv .git/logs actual_logs &&
+	cmd //c "mklink /D .git\logs ..\actual_logs" &&
+	git rebase -f HEAD^ &&
+	test -L .git/logs &&
+	rm .git/logs &&
+	mv actual_logs .git/logs
+'
+# Note: `test -L` will fail if ".git/logs" is not a symlink OR if ".git/logs"
```

Comment:
IMHO we do not need this code comment (it would be the first such code comment in t3400, I believe).

Reply:
I think comments are most of the time useful but you're the boss here so I removed them 😉